app/src/main/java/ch/epfl/sdp/appart/user/UniversityEmailDatabase.java

```diff
@@ -0,0 +1,32 @@
+package ch.epfl.sdp.appart.user;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+
+/**
+ * The functionality of this class is to store a database of
+ * accepted and trusted university email providers.
+ */
+public class UniversityEmailDatabase {
```

Comment:
Could we set the attributes and the method in static ? This way we don't have to create a UniversityDataBase in the class above.